tests/TestCase/Panel/SqlLogPanelTest.php

```diff
@@ -25,13 +25,6 @@
  */
 class SqlLogPanelTest extends TestCase
 {
-    /**
-     * fixtures.
-     *
-     * @var array
-     */
-    public $fixtures = ['core.Articles'];
```

Comment:
Do we need the records in this fixture?

Reply:
Yes there are tests using Articles table/fixture.